deps/wazuh_testing/wazuh_testing/vulnerability_detector.py

```diff
@@ -287,3 +329,109 @@ def modify_system(os_name="CentOS Linux", os_major="7", name="centos7", agent_id
     query_string = f'update AGENT set OS_NAME="{os_name}", OS_MAJOR="{os_major}", OS_MINOR="{os_minor}",\
                      NAME="{name}" WHERE id="{agent_id}"'
     make_query(GLOBAL_DB_PATH, [query_string])
+
+
+def get_num_vulnerabilities():
+    """
+    Get the number of vulnerabilities inserted in VULNERABILITIES table of CVE DB.
+    """
```

Comment:
Same, it should fit on one line.

Reply:
Done in bc869705500dbe0aa3575b3aafd58be9507220d4